lib/modules/manager/terraform/extractors/others/modules.ts

```diff
@@ -90,15 +90,11 @@ export class ModuleExtractor extends DependencyExtractor {
       dep.currentValue = azureDevOpsSshRefMatch.groups.tag;
       dep.datasource = GitTagsDatasource.id;
     } else if (gitTagsRefMatch?.groups) {
-      if (gitTagsRefMatch.groups.path.includes('//')) {
+      if (gitTagsRefMatch.groups.subfolder) {
         logger.debug('Terraform module contains subdirectory');
```

Comment:
Is the subfolder otherwise unimportant and ignored except for this message?

Reply:
no, after change, to check subdirectory cases, capture the group "subfolder" over checking has "//" 
https://regexr.com/8enfs

besides no difference between module has subdir or not one to set dep and package names